tmail-backend/jmap/extensions-es-v7/src/main/java/com/linagora/tmail/james/jmap/ESEmailAddressContactSearchEngine.java

```diff
@@ -74,6 +70,6 @@ public ESEmailAddressContactSearchEngine(ReactorElasticSearchClient client, Elas
             .map(Arrays::asList)
             .flatMapIterable(Function.identity())
             .map(Throwing.function(hit -> new EmailAddressContact(UUID.fromString((String) hit.getSourceAsMap().get("id")),
-                new MailAddress((String) hit.getSourceAsMap().get("address")), "", "")));
+                new ContactFields(new MailAddress((String) hit.getSourceAsMap().get("address")), "", ""))));
```

Comment:
I see we set the default value for firstname and surname. We can omit them at this moment?

Not a big deal though.

Reply:
Nope... I set the default values but it's in a scala class. This is Java though, you seem to need to declare them, the conversion does not work. If I omit them, I can't compile :)